project/dexter-monitor/public/ctrl/currentCtrl.js

```diff
@@ -25,6 +25,63 @@
  */
 "use strict";
 
-monitorApp.controller("CurrentCtrl", function() {
+monitorApp.controller("CurrentCtrl", function($scope, $http, $log, ProjectService, uiGridConstants) {
```

Comment:
Please use a name that shows its purpose or intention.
for example, CurrentCtrl -> CurrentDefectStatusCtrl, AllCurrentStatusCtrl, AllOverviewStatusCtrl, or etc.